pubsub/cloud-client/quickstart/sub.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START pubsub_quickstart_sub_all]
+import argparse
+import time
+# [START pubsub_quickstart_sub_deps]
+from google.cloud import pubsub_v1
+# [END pubsub_quickstart_sub_deps]
+
+
+def sub(project_id, subscription_name):
+    """Receives messages from a Pub/Sub subscription."""
+    # [START pubsub_quickstart_sub_client]
+    # Initializes the Subscriber client
+    client = pubsub_v1.SubscriberClient()
+    # [END pubsub_quickstart_sub_client]
+    # Creates a fully qualified identifier in the form of
+    # `projects/{project_id}/subscriptions/{subscription_name}`
+    subscription_path = client.subscription_path(
+        project_id, subscription_name)
+
+    def callback(message):
+        print('Received message {} of message ID {}'.format(
+            message, message.message_id))
+        # Acknowledges the message. Unack'ed messages will be redelivered.
+        message.ack()
+        print('Acknolwedged message of message ID {}\n'.format(
+            message.message_id))
+
+    client.subscribe(subscription_path, callback=callback)
+    print('Listening for messages on {}..\n'.format(subscription_path))
+
+    # The subscriber is non-blocking. We keep the main thread from exiting
```

Comment:
Non-blocking doesn't convey to me that it actually starts up a background thread to wait for messages.

Also, nit: In general, don't use "We" in our technical writing. Tell the developer what to do actively.

Prefer:

Keep the main thread from exiting so that the subscriber can process messages in the background.

("asynchronously in the background" seems a little redundant to me.)

Reply:
Done.